schemas/user/company_info.schema_v1.0.json

```diff
@@ -0,0 +1,164 @@
+        "Senegalese", "Serbian", "Citizen of Seychelles", "Sierra Leonean", "Singaporean",
+        "Slovak", "Slovenian", "Solomon Islander", "Somali", "South African", "South Korean",
+        "South Sudanese", "Spanish", "Sri Lankan", "St Helenian", "St Lucian", "Stateless",
+        "Sudanese", "Surinamese", "Swazi", "Swedish", "Swiss", "Syrian", "Taiwanese", "Tajik",
+        "Tanzanian", "Thai", "Togolese", "Tongan", "Trinidadian", "Tristanian", "Tunisian",
+        "Turkish", "Turkmen", "Turks and Caicos Islander", "Tuvaluan", "Ugandan", "Ukrainian",
+        "Uruguayan", "Uzbek", "Vatican citizen", "Citizen of Vanuatu", "Venezuelan", "Vietnamese",
+        "Vincentian", "Wallisian", "Welsh", "Yemeni", "Zambian", "Zimbabwean"
+      ]
+    }
+  },
+  "properties": {
+    "legalBusinessName": {
+      "description": "Legal business name",
+      "type": "string",
+      "maxLength": 120,
+      "pattern": "^[a-zA-Z0-9 ]*$"
+    },
+    "businessName": {
+      "description": "Business name- company's operating name if it's different than the legal business name",
+      "type": "string",
+      "maxLength": 120,
+      "pattern": "^[a-zA-Z0-9 ]*$"
+    },
+    "businessRegisteredCountry": {
+      "description": "The country where the business is registered",
+      "type": "string",
+      "$ref": "#/definitions/country"
+    },
+    "registeredBusinessAddress": {
+      "$ref": "#/definitions/address",
+      "description": "Address of the company's registered office"
+    },
+    "registrationNumberIncorporation": {
+      "description": "Registration number of incorporation",
+      "type": "string",
+      "maxLength": 30
+    },
+    "companiesHouseRegistrationNumber": {
+      "description": "Companies House Registration Number (CRN) for UK country only",
+      "type": "string",
+      "maxLength": 8
+    },
+    "employerIdentificationNumber": {
+      "description": "Employer Identification Number (EIN) for US country only",
+      "type": "string",
+      "maxLength": 10
+    },
+    "beneficialOwners": {
+      "description": "List of beneficial owners",
+      "type": "array",
+      "items": { "$ref": "#/definitions/beneficialOwner" },
+      "minItems": 1
+    },
+    "industryCode": {
+      "description": "Merchant category codes (MCC)",
+      "type": "string",
+      "maxLength": 4,
+      "pattern": "^[0-9]{4}$"
+    },
+    "vatNumber": {
+      "description": "VAT number",
+      "type": "string",
+      "maxLength": 15
+    },
+    "businessWebsite": {
+      "description": "Business website",
+      "type": "string",
+      "maxLength": 100,
+      "format": "uri"
+    },
+    "statementDescriptor": {
+      "description": "Statement descriptor explains charges or payments on customer's bank statements",
+      "type": "string",
+      "minLength": 5,
+      "maxLength": 22,
+      "pattern": "^[^<>, '\"*]*$"
+    }
+  },
+  "required": [
+    "name",
```

Comment:
Does company have a "name" field?

Reply:
company does not have name field , it should be legalBusinessName
@BarAzulay

Reply:
no, this is related to `beneficialOwner`. good catch.